source/languages/en/riak/community/product-advisories/210-dataloss.md

```diff
@@ -60,7 +60,7 @@ Then configure handoff.ip in riak.conf to an external IP address or 0.0.0.0 on a
 
 Perform a rolling restart of Riak across your cluster to activate the new setting.
 
-After correcting the configuration and restarting the nodes, you should run Riak KV repair on each cluster member as documented at [http://docs.basho.com/riak/latest/ops/running/recovery/repairing-partitions/](http://docs.basho.com/riak/latest/ops/running/recovery/repairing-partitions/) to recreate any missing replicas from available replicas elsewhere in the cluster.  It is recommended to perform the Riak KV repair in a round-robin fashion on each node of your cluster (node0, node1, node2, etc). Repeat this round-robin repair “n_val - 1” times. For example: the default configuration for n_val is 3, which means you would run Riak KV repair twice across the entire cluster. 
+After correcting the configuration and restarting the nodes, you should run Riak KV repair on each cluster member as documented at [http://docs.basho.com/riak/latest/ops/running/recovery/repairing-partitions/](http://docs.basho.com/riak/latest/ops/running/recovery/repairing-partitions/) to recreate any missing replicas from available replicas elsewhere in the cluster.  We recommend performing the Riak KV repair in a round-robin fashion on each node of your cluster (node0, node1, node2, etc). Repeat this round-robin repair “n_val - 1” times. For example: the default configuration for n_val is 3, which means you would run Riak KV repair twice across the entire cluster. 
```

Comment:
1.) I know this PR has been here forever, so I'm sorry to be just getting to it. 
2.) I feel like there has got to be a better way to refer to the repair action. I wonder if calling it the `repair` action would work?